tests/test_flows/test_flow_functions.py

```diff
@@ -221,3 +225,13 @@ def test_are_flows_equal_ignore_if_older(self):
         self.assertRaises(ValueError, assert_flows_equal, flow, new_flow,
                           ignore_parameter_values_on_older_children=flow_upload_date)
         assert_flows_equal(flow, flow, ignore_parameter_values_on_older_children=None)
+
+    @unittest.skipIf(LooseVersion(sklearn.__version__) < "0.20",
+                     reason="OrdinalEncoder introduced in 0.20. "
+                            "No known models with list of lists parameters in older versions.")
+    def test_sklearn_to_flow_list_of_lists(self):
+        from sklearn.preprocessing import OrdinalEncoder
+        ordinal_encoder = OrdinalEncoder(categories=[[0, 1], [0, 1]])
+        flow = openml.flows.sklearn_to_flow(ordinal_encoder)
+        self._add_sentinel_to_flow_name(flow)
+        flow.publish()
```

Comment:
Why do you need to publish this flow if the error happens in `sklearn_to_flow`?

Reply:
I thought it might be useful to check the serialized flow is also valid for the server. You're right that it isn't strictly necessary. Should I remove it?

Reply:
I think it is a good idea to ask the server for its opinion, but you should then also download the flow again and compare whether the deserialization works correctly.

Reply:
Done.